src/pages/preparing-for-tech-talk-part-1-motivation/index.es.md

```diff
@@ -78,8 +78,8 @@ Así que si quieres dar una charla, *hablar* con gente es una buena forma de emp
 
 ---
 
-En este post he descrito el marco de referencia que me resulta útil para pensar en ideas para charlas. De nuevo, quiero enfatizar que solo estoy compartiendo lo que funciona para mi, hay muchos [tipos de charlas](https://mobile.twitter.com/jackiehluo/status/1077717283026411520) y tu mirada en esto puede ser muy diferente.
+En este artículo he descrito el marco de referencia que me resulta útil para pensar en ideas para charlas. De nuevo, quiero enfatizar que solo estoy compartiendo lo que funciona para mi, hay muchos [tipos de charlas](https://mobile.twitter.com/jackiehluo/status/1077717283026411520) y tu mirada en esto puede ser muy diferente.
 
-En el siguiente artículo de esta serie, hablaré sobre la preparación del bosquejo de la charla, las diapositivas, el ensayo de las charlas y lo que hago el día de la presentación.
+En los siguientes artículos de esta serie, hablaré sobre la preparación del bosquejo de la charla, las diapositivas, el ensayo de las charlas y lo que hago el día de la presentación.
 
-**A continuación en esta serie: [Preparándose para una charla técnica, Parte 2: Qué, Por Qué y Cómo](/preparing-for-tech-talk-part-2-what-why-and-how/)**
+**A continuación en esta serie: [Preparándose para una charla técnica, Parte 2: Qué, Por Qué y Cómo](/es/preparing-for-tech-talk-part-2-what-why-and-how/)**
```

Comment:
I'd like to keep links pointing to English URLs. We can later make this automatic.